src/client/java/io/wispforest/limelight/impl/ui/ResultEntryComponent.java

```diff
@@ -126,24 +138,32 @@ public boolean onKeyPress(int keyCode, int scanCode, int modifiers) {
 
     @Override
     public boolean onCharTyped(char chr, int modifiers) {
-        root().focusHandler().focus(screen.searchBox, FocusSource.MOUSE_CLICK);
-        screen.searchBox.onCharTyped(chr, modifiers);
-
-        return true;
+        if (root() != null) {
```

Comment:
why would `root()` be null here? aren't input events only routed to components if they're mounted?

Reply:
> why would root() be null here?
IDK. I just now that it can, because it crashed. I'll see if this still happens after my aforementioned changes